plugins/core/logreport.go

```diff
@@ -37,6 +39,15 @@ type logTracingContext interface {
 	GetEndPointName() string
 }
 
+type logTracingSpan interface {
```

Comment:
Why do we need to add this kind of span? It will make the agent more complex.

Reply:
If you just want to get the endpoint name and parent span, they have already defined in the TracingSpan.